src/ui/server/controllers/insurance/business/company-details/index.POST-continue.test.ts

```diff
@@ -88,6 +90,25 @@ describe('controllers/insurance/business/companies-details', () => {
 
         expect(res.redirect).toHaveBeenCalledWith(NATURE_OF_BUSINESS);
       });
+
+      it('should call mapAndSave.companyDetails once with updateBody and application', async () => {
+        req.body = {
+          [INPUT]: '8989898',
+          [TRADING_NAME]: 'true',
+          [TRADING_ADDRESS]: 'false',
+          [PHONE_NUMBER]: VALID_PHONE_NUMBERS.LANDLINE,
+        };
+
+        await post(req, res);
+
+        expect(mapAndSave.companyDetails).toHaveBeenCalledTimes(1);
+
+        const updateBody = {
+          ...req.body,
+          ...mockCompany,
+        };
+        expect(mapAndSave.companyDetails).toHaveBeenCalledWith(updateBody, mockApplication);
+      });
```

Comment:
Great stuff, can we add some tests for error handling:  https://github.com/UK-Export-Finance/exip/blob/main-application/src/ui/server/controllers/insurance/policy-and-export/about-goods-or-services/index.test.ts#L330